room/templates/site_list_one.html

```diff
@@ -6,7 +6,25 @@
   | View a Site
 {% endblock %}
 {%block about_this_page %}
-<h2>Here's site # {{ entries[0].number }}</h2>
+<h2>Site Number <input type="text"
```

Comment:
I think he wanted this input to be on the header of *every* page.

Reply:
well then